src/shared/rpc/methods/sign-psbt.ts

```diff
@@ -2,17 +2,19 @@ import { DefineRpcMethod, RpcRequest, RpcResponse } from '@btckit/types';
 import * as yup from 'yup';
 
 import { networkModes } from '@shared/constants';
+import { isNumber, isUndefined } from '@shared/utils';
+
+function testIsNumberOrArrayOfNumbers(value: unknown) {
+  if (isUndefined(value)) return true;
```

Comment:
we are ok with undefined value here?

Reply:
yeah we need to allow undefined as an okay case, otherwise it fails it